packages/flutter/lib/src/cupertino/app.dart

```diff
@@ -243,7 +243,7 @@ class _CupertinoAppState extends State<CupertinoApp> {
       _navigatorObservers = List<NavigatorObserver>.from(widget.navigatorObservers)
         ..add(_heroController);
     } else {
-      _navigatorObservers = null;
+      _navigatorObservers = const <NavigatorObserver>[];
```

Comment:
is this an unrelated change?

Reply:
Fixes a bug the demo revealed. The observer list can't be null.